nutils/solver.py

```diff
@@ -323,7 +322,7 @@ def solve_linear(target, residual:integraltuple, *, constrain:arrayordict=None,
   ----------
   target : :class:`str`
       Name of the target: a :class:`nutils.function.Argument` in ``residual``.
-  residual : :class:`nutils.sample.Integral`
+  residual : :class:`nutils.function.Array` or :class:`nutils.evaluable.Array`
```

Comment:
same here, can we use the `AsEvaluableArray` protocol in these situations? (depends on sphinx I guess)

Reply:
That works.